python/README.rst

```diff
@@ -58,6 +66,244 @@ Installation
+
+
+Orbital Element Transformation
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Function ``pyoorb.oorb_element_transformation`` provides
+transformations between different orbital element
+definitions. Required parameters are ``in_orbits`` (an orbit array as
+defined above) and ``in_element_type`` (the orbital element type
+integer code: ``CART``: 1, ``COM``: 2, ``KEP``: 3). The function
+outputs the orbit array as defined above using the element scheme
+defined through ``in_element_type``, as well as the corresponding
+error code.
+
+The following example transforms the orbit array defined above from a
+Keplerian to a cartesian definition:
+
+    >>> new_orbits, err = oo.pyoorb.oorb_element_transformation(
+    ...     in_orbits=orbits,
+    ...     in_element_type=1)
+    >>> print(err)
+    0
+    >>> print(new_orbits)
+    [[ 0.00000000e+00 -1.13248995e+00 -1.21090780e-01 -1.80398368e-01
+       5.81046365e-03 -1.61918824e-02  3.98214412e-03  1.00000000e+00
+       5.15445000e+04  1.00000000e+00  1.25000000e+01  1.50000000e-01]
+                                     ...
+     [ 2.00000000e+00 -6.02921087e-01 -5.37976414e-01 -6.83341536e-03
+       1.03814468e-02 -2.24243823e-02 -1.46645896e-04  1.00000000e+00
+       5.15445000e+04  1.00000000e+00  2.05000000e+01  1.50000000e-01]]    
+
+The definitions of the individual columns are provided above.
+
+Orbit Propagation
+^^^^^^^^^^^^^^^^^
+
+The function ``pyoorb.oorb_propagation`` propagates one or more orbits
+(``in_orbits``) to a desired epoch (``in_epoch``). The epoch has to be
+provided as an array with ``dtype=np.double, order='F'`` (see example
+below) and has to contain two elements: the epoch as modified Julian
+date and the corresponding timescale type (``UTC``: 1, ``UT1``: 2,
+``TT``: 3, ``TAI``: 4). The user can decide between an N-body
+integration (``in_dynmodel='N'``) or a faster but less accurate
+two-body integration (``in_dynmodel='2'``).
+
+The following example creates a target epoch and propagates all three
+orbits defined above using an N-body integration:
+
+    >>> epoch = np.array([51232.23, 3], dtype=np.double, order='F')
+    >>> orb, err = oo.pyoorb.oorb_propagation(in_orbits=orbits,
+    ...                                       in_epoch=epoch,
+    ...                                       in_dynmodel='N')
+    >>> print(err)
+    0
+    >>> print(orb)
+    [[0.00000000e+00 1.46902259e+00 3.34239111e-01 2.49612224e-01
+      3.91849109e+00 4.81057800e-01 2.64937688e+00 3.00000000e+00
+      5.12322300e+04 3.00000000e+00 0.00000000e+00 0.00000000e+00]
+     [1.00000000e+00 2.49303082e+00 7.89455978e-01 1.39563343e-01
+      2.17499921e+00 4.66102654e+00 3.83095531e+00 3.00000000e+00
+      5.12322300e+04 3.00000000e+00 0.00000000e+00 0.00000000e+00]
+     [2.00000000e+00 2.43234703e+00 7.01646868e-01 9.21820573e-03
+      5.85002081e+00 3.58026888e+00 4.96413849e+00 3.00000000e+00
+      5.12322300e+04 3.00000000e+00 0.00000000e+00 0.00000000e+00]]
```

Comment:
Have you noticed a bug with the returned propagated orbits (looks like it's there above too) -- there are blank values for H and g).

Reply:
fixed!